distribution/pom.xml

```diff
@@ -108,6 +108,8 @@
                                 <argument>io.druid.extensions:druid-stats</argument>
                                 <argument>-c</argument>
                                 <argument>io.druid.extensions:druid-examples</argument>
+                                <argument>-c</argument>
+                                <argument>io.druid.extensions:sample-client-sslcontext</argument>
```

Comment:
I guess TrustManager should take care of the isolation but not sure about the details of it.